component/identifiable.go

```diff
@@ -9,12 +9,13 @@ import (
 	"strings"
 )
 
-// typeAndNameSeparator is the separator that is used between type and name in type/name composite keys.
-const typeAndNameSeparator = "/"
+// TypeAndNameSeparator is the separator that is used between type and name in type/name composite keys.
+const TypeAndNameSeparator = "/"
```

Comment:
Maybe this should be renamed since we also use it for separating `DataType` and pipeline name. Alternatively, move it to internal and make it private